packages/static-wado-webserver/lib/index.mjs

```diff
@@ -48,14 +52,33 @@ const gzipHeaders = (res, path) => {
   }
 };
 
+const addQueryCall = async (router, level, params, key) => {
+  const name = params[key];
+  if (!name) return;
+  const plugin = await importPlugin(name);
```

Comment:
importPlugin can throw and error. We should wrap this block with try/catch

Reply:
Do you have a suggestion for how to handle this other than stopping execution/loading of the program?  Or do you just want a better error message followed by exiting the program?